test/e2e/authprovider/cypress/run-provider-login-cypress.sh

```
@@ -0,0 +1,109 @@
+    exit 1
+  fi
+}
+
+ensure_cypress_installed
+
+
+if command -v ss >/dev/null 2>&1; then
+  PIDS="$(ss -ltnp "sport = :${CALLBACK_PORT}" 2>/dev/null | sed -n 's/.*pid=\([0-9][0-9]*\).*/\1/p' | sort -u)"
+  if [[ -n "$PIDS" ]]; then
+    echo "Killing stale process(es) on ${CALLBACK_PORT}: $PIDS" >&2
+    kill $PIDS || true
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)